google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -515,6 +563,30 @@ private Builder() {
                               .build())
                       .build());
 
+      long numOfChannels = 2 * Runtime.getRuntime().availableProcessors();
+      bulkReadRowsSettings =
+          BigtableBulkReadRowsCallSettings.newBuilder(new ReadRowsBatchingDescriptor())
+              .setRetryableCodes(readRowSettings.getRetryableCodes())
+              .setRetrySettings(
+                  readRowSettings
+                      .getRetrySettings()
+                      .toBuilder()
+                      .setMaxRpcTimeout(Duration.ofSeconds(300))
+                      .setTotalTimeout(Duration.ofMinutes(20))
+                      .build())
+              .setBatchingSettings(
+                  BatchingSettings.newBuilder()
+                      .setIsEnabled(true)
+                      .setElementCountThreshold(100L)
+                      .setRequestByteThreshold(400L * 1024L)
+                      .setDelayThreshold(Duration.ofSeconds(1))
+                      .setFlowControlSettings(
+                          FlowControlSettings.newBuilder()
+                              .setLimitExceededBehavior(LimitExceededBehavior.Block)
+                              .setMaxOutstandingElementCount(10L * 100L * numOfChannels)
```

Comment:
please extract 100 to a local variable to make it obvious that its shared with element count threshold

long maxElementPerBatch = 100L;